controllers/authentication/openidconnect_controller.go

```diff
@@ -67,8 +77,29 @@ func (r *OpenIDConnectReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 		algs = append(algs, string(alg))
 	}
 
+	var caBundle dynamiccertificates.CAContentProvider
+	if config.Spec.CABundle != nil {
+		caBundle, err = dynamiccertificates.NewStaticCAContent("CABundle", config.Spec.CABundle)
+		if err != nil {
+			log.Info("Invalid CABundle")
```

Comment:
Here the reconciliation should end and not return any error - we do not want to constantly reconcile when the `config.Spec.CABundle` stays the same.

Reply:
Log error here